### PR TITLE
resnest.py, nn/dropblock.py & nn/splat.py  for MXNet-master

### PR DESCRIPTION
**Pull request by jinboci**
resnest.py nn/dropblock.py & nn/splat.py for MXNet-master
Passed tests for `resnest.py` at `test_imagenet_models()` in `test_model_zoo.py` on my local sever.

Changes due to mxnet-master's different implementations from 1.7:
- `nn.HybridSequential` on mxnet-master does not accept the parameter `prefix`
- No `name_scope` defined in class `HybridSequential` `HybridBlock` `Block` on mxnet-master (block.py)

Latency operators still in use:
- `AdaptiveAvgPooling2D`

### CODE REVIEW

File: gluoncv/model_zoo/resnest.py [hunk 1]
```diff
@@ -438,15 +440,12 @@ def resnest50(pretrained=False, root='~/.mxnet/models', ctx=cpu(0), **kwargs):
                     deep_stem=True, avg_down=True,
                     avd=True, avd_first=False,
                     use_splat=True, dropblock_prob=0.1,
-                    name_prefix='resnest_', **kwargs)
-    if pretrained:
-        from .model_store import get_model_file
-        model.load_parameters(get_model_file('resnest50', root=root), ctx=ctx)
```
Comment (jinboci) [inline gluoncv/model_zoo/resnest.py:444]: `model.load_parameters` deleted？

**Comment by mli** [comment]
Job PR-1506-6 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-1506/6/index.html 
Code coverage of this PR:  vs. Master:

**Comment by chinakook** [comment]
There are an existing c++ implementation of groupnorm and it's correct and fast as pytorch after this fix https://github.com/apache/incubator-mxnet/pull/18199 as I've tested.

**Comment by jinboci** [comment]
Changes have already been merged

**Pull request closed without merging**